src/draw/list_view.rs

```diff
@@ -100,14 +108,19 @@ where
             (self.selected_item - self.skip_offset, self.skip_offset)
```

Comment:
Not sure whether it's related but this piece of code above gives a panic now if running in debug build. So this code should work:

```rust
let displayed_items = (((space.height - margin.top - margin.bottom + item_spacing)
    / (entry_height + item_spacing)) as usize)
    .saturating_sub(has_subname as usize);

let max_offset = self.skip_offset + displayed_items;
let (selected_item, skip_offset) = if self.selected_item < self.skip_offset {
    (0, self.selected_item)
} else if max_offset <= self.selected_item {
    (
        displayed_items.saturating_sub(1),
        self.skip_offset + (self.selected_item - max_offset) + 1,
    )
} else {
    (self.selected_item - self.skip_offset, self.skip_offset)
};
```